client/src/document/on-github.tsx

```diff
@@ -62,7 +63,10 @@ MDN URL: https://developer.mozilla.org$PATHNAME
 * Folder: \`$FOLDER\`
 * MDN URL: https://developer.mozilla.org$PATHNAME
 * GitHub URL: $GITHUB_URL
+* Last commit hash: \`$HASH\`
 * Document last modified: $DATE
+* Branch: \`$BRANCH\`
```

Comment:
This will always be `main`. (local dev doesn't matter) And if it's not the usual `main` embedded in the `GitHub URL` above. Let's remove it.

Reply:
Yes, it will. I included it initially as a demonstration of how easily anyone could add more links from the "building blocks". Since we are generating all URLs in Node, this line is no longer relevant.